test/integration/cli-nfs-shared-volumes-create-failure.test.js

```diff
@@ -0,0 +1,149 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright (c) 2017, Joyent, Inc.
+ */
+
+//
+// Summary:
+//
+// These tests ensure that when provisioning fails for a volapi storage VM, we
+// get an error message from sdc-docker rather than a message telling us the
+// creation was successful. We "break" provisioning for the 10g package by
```

Comment:
Adding the ticket is a good idea. I'll do that.

And yeah, I don't have a better way atm for ensuring the provision fails. The runtest(s) scripts do ensure that we've indicated that this system is not a production system at least.